packages/os/os.spec

```diff
@@ -232,6 +236,11 @@ Summary: Settings generator for ECS
 Summary: Dynamic setting generator for kubernetes
 %description -n %{_cross_os}pluto
 %{summary}.
+
+%package -n %{_cross_os}hachiko
+Summary: Autoscaling warmpool support
+%description -n %{_cross_os}hachiko
+%{summary}.
 %endif
```

Comment:
The conditionals to add packages depending on the variant being built, change, you need to rebase `develop` to bring those changes.